src/lib.rs

```diff
@@ -127,6 +127,14 @@ define_signed_integers! {
     i64_le i64_be: 8 i64,
     i128_le i128_be: 16 i128,
 }
+impl_try_into_ptr_size!(TryInto<isize> for i16_le via i16);
```

Comment:
I think these can be merged into the `impl_signed_integer_traits` macro instead of being defined separately.